### ISSUE TITLE
No settings upgrade

### ISSUE THREAD

**Issue by LoganDark** [nickgravgaard/ElasticNotepad#11]
Recently jumped a couple versions forward and noticed that the new settings weren't there until I clicked "revert to default". I think the app should automatically add any missing settings if they aren't found.

**Comment by nickgravgaard** [comment]
Yeah, I don't think I ever considered the case for when we add a setting in a new version of the app but the user has an config file generated by an older version. I'll have a think about this.
